week-1/implement/max.js

```diff
@@ -0,0 +1,16 @@
+function max(array) {
+  let tempArray
+  switch (true) {
+    case array.length === 0:
```

Comment:
is not really ideal to use switch statement for sure thing, plus is not idea to pass `true` in the switch param. No need to switch for such sernario. 
Switch statements are useful when using different types of blocks of code, in this case you don't have different cases or different blocks of code is just one array so you don't need to use witch.

can you try a different way please? if you need more help or hints let me know!